docs/content/contribute/model-engineering-meeting/_index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: OSCAL Model Engineering Meeting
+description: Details about the OSCAL Model Engineering Meeting
+layout: devlunch
+aliases:
+- /contribute/devlunch/
+- /contribute/dev-lunch/
+toc:
+  enabled: true
+  headingselectors: "h3,h4"
+weight: 10
+---
+
+{{% usa-intro %}}The OSCAL Model Engineering Meeting is a one-hour teleconference every other week to discuss OSCAL models, how they are developed, and development efforts by NIST staff and the greater OSCAL community using them.{{% /usa-intro %}}
+
+Note: (10/20/2021) The Bluejeans info below has changed.
+
+{{% usa-tag %}}Day and Time{{% /usa-tag %}} Every other Thursday at 10 AM ET
```

Comment:
Should a starting day be specified? Is it February as well?

Reply:
Yes, I implied that in the comment on the PR, I should have been more clear. Dave replied for this one it should be January 26th.

Reply:
Done, added to updated commit.